tools/json-api-client/tsconfig.json

```diff
@@ -0,0 +1,21 @@
+{
+  "compilerOptions": {
+    "composite": true,
+    "declaration": true,
+    "declarationMap": true,
+    "strict": true,
+    "skipLibCheck": true,
```

Comment:
Can we remove this?

Reply:
Don't we need this for `global` & `json-api-normalizer`?

Reply:
We shouldn't no, this is a relic of having our yarn workspaces mashed up.